IS05Utils.py

```diff
@@ -604,7 +604,7 @@ def park_resource(self, resource_type, resource_id):
         valid, response = self.checkCleanRequestJSON("PATCH", url, data=data)
         if valid:
             try:
-                staged_params = response['transport_params']
+                response.get('transport_params')
```

Comment:
Personally I think this is over-aggressive linting that actually reduces the code readability. :-(

Reply:
We can disable some linting if needed (probably in individual cases too), but there are lots of cases where an unused variable is an indication of a bug.

Reply:
True. Sigh.